src/selectors/blockchain.js

```diff
@@ -1,9 +1,13 @@
-import { get } from 'lodash'
+import Decimal from 'decimal.js'
 
 export const selector = state => state.blockchain
 
 export const getDefaultAccount = state => selector(state).defaultAccount
 
+export const getGasCosts = state => selector(state).gasCosts
+
+export const getGasPrice = state => new Decimal(parseInt(selector(state).gasPrice, 10))
```

Comment:
Same here as above, I'd prefer it if we returned undefined here, for gas prices that have not been set yet